CHANGELOG.md

```diff
@@ -34,26 +34,27 @@ _None._
 
 ### Breaking Changes
 
-- `nvm_install` utility is removed. See [MIGRATION.md](./MIGRATION.md#from-200-to-300) for details.
+- `nvm_install` utility is removed. See [MIGRATION.md](./MIGRATION.md#from-200-to-300) for details. [#73]
 
 ### New Features
 
 _None._
 
 ### Bug Fixes
 
-_None._
+ - Fix issue in `annotate_test_failures` where the error details' XML node could sometimes be `nil`. [#76]
+ - Make `install_swiftpm_dependencies` use `-onlyUsePackageVersionsFromResolvedFile` flag, to prevent `Package.resolved` from being modified on CI. [#75]
```

Comment:
Note: I forgot to add the CHANGELOG entry in #75 so I took the opportunity of this PR to add it here as well.